setup.py

```diff
@@ -27,9 +27,9 @@ def get_contents(filename):
     #packages:
     packages=["uiplib"],
 
-    license="LICENSE.txt",
+    license="LICENSE",
 
-    url="example.com",
+    url="github.com/NIT-dgp/UIP",
 
```

Comment:
I know that it's obvious but still I think It will be better to add comments on top of both license and url since all other line have comments even though they are obvious :stuck_out_tongue: